install/0000_30_machine-api-operator_08_baremetalhost.crd.yaml

```diff
@@ -705,6 +709,7 @@ spec:
                     type: string
                 type: object
             required:
+            - errorCount
```

Comment:
Is this field being required going to be a problem for upgrades? i.e. does it mean that after updating the CRD, the baremetal-operator won't be able to write to the status until it is also upgraded?
If so this should probably be fixed upstream.

Reply:
As discussed, I've added a default value for the field to allow smoother upgrade